### ISSUE TITLE
Crash upon creating new world with test 8 and Forge 10.13.2.1307

### ISSUE THREAD

**Issue by NotGyro** [CannibalVox/DimDoors#21]
I don't know if this is a known issue or not, but on generating / loading / whatever a new world for the first time I got a null pointer error and Minecraft crashed to desktop. The weird thing is: I had just been running exactly the same set of mods & version of Forge, finding gateways, and walking through them, with no problem. I even spotted a worldgen rift. It worked just fine, for some reason. The world seed was 6985892767528132485 and it was a Biomes O' Plenty world.

```
java.lang.NullPointerException: Exception in server tick loop
	at StevenDimDoors.mod_pocketDim.world.PocketProvider.func_76556_a(PocketProvider.java:92)
	at net.minecraft.world.WorldProvider.func_76558_a(WorldProvider.java:45)
	at net.minecraft.world.World.<init>(World.java:244)
...
```

The full crashlog is here: http://pastebin.com/6c0FFG23

**Comment by DPxKnightxZero** [comment]
I would like you to test this using this version of forge

**Comment by chaoskagami** [comment]
No offense to you DPxKnightxZero, but you should really link to the official forge page with what release you want them to use. I wouldn't download a random jar off a mega link. It could be ANYTHING. I'll let you off the hook since the SHA512 matches the one from forge and it's not been tampered with, but I'd be careful with that in the future.

Also, you should be aware that bugfix version of forge is old as of two days ago. http://files.minecraftforge.net/maven/net/minecraftforge/forge/1.7.10-10.13.2.1342-1.7.10/forge-1.7.10-10.13.2.1342-1.7.10-installer.jar is now the latest, but I highly doubt the forge version is his issue. More than likely this is some kind of compatibility issue, but his logs are incomplete.

Considering you're on linux, crash it again, go to ~/.minecraft/logs and upload the fml-client-latest.log and latest.log somewhere where people can read them. The launcher only shows a stack dump and that's not good enough.

**Comment by DPxKnightxZero** [comment]
Oh double thanks :D also for everyone information I started learning java

Class thankyou
   public static void main(string args[]){
System.out.println("I will make sure in the future to use forge direct links :)")
}

**Comment by DPxKnightxZero** [comment]
Okay this is interesting one of my server members has this but has no issues prior ... https://cdn.fbsbx.com/hphotos-xta1/v/t59.2708-21/11095957_10206636476379136_334686206_n.zip/fml-client-latest.zip?oh=886801d0f3c7dbfcc73ea0b26e0b516e&oe=5515B10F&dl=1

**Comment by chaoskagami** [comment]
DP, you have a completely different set of mods listed there than Gyro. If it's the same issue though, the only thing that I do know is that it isn't likely to be DD causing this. Try to get down to the bare minimum of mods to reproduce this. Move every mod out of the mods folder, and move DD in. Continuously add everything back in one by one until it crashes. Whatever was moved in last is the cause.

(I don't even know if it's okay to help out with the bug reports since this is Vox's mod. The last thing I want to do is be a backseat moderator, since that annoys everybody. I've taken a good read through DD's source to know what I'm talking about, but still.)

**Comment by DPxKnightxZero** [comment]
Its not annoying its great and I already decided untill DD is fixed where I don't get ticking doors crash and I don't have dimensional ID conflicts I am just going to wait.

**Comment by AltonV** [comment]
I got the same crash but only once.
I created a creative superflat map with the preset tunnelers dream when the crash happened.
The map loaded fine after starting minecraft again.
After that I created several maps with no problem including a map with the same settings as the first (except the seed). They generated fine without crashing.

http://pastebin.com/HqJdpw6S

**Comment by CannibalVox** [comment]
Fixed with 27d17d86a148721058f4dd092707e1f59c77eb4f
